subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -125,11 +136,11 @@ private ExceptionStyle getShowStackTraceOption() {
         }
     }
 
-    private FailureDetails constructFailureDetails(String granularity, Throwable failure) {
+    private FailureDetails constructFailureDetails(String granularity, Throwable failure, boolean gradleEnterprisePluginApplied) {
```

Comment:
To avoid spreading this param around, I'd consider passing in some kind of more general strategy object here that can populate the resolution details. Your call though.

Reply:
Yep, good suggestion.  Done.